CONTRIBUTING.md

```diff
@@ -135,3 +135,46 @@ available at [http://contributor-covenant.org/version/1/4][version]
 
 [homepage]: http://contributor-covenant.org
 [version]: http://contributor-covenant.org/version/1/4/
+[version]: http://contributor-covenant.org/version/1/4/
+
+### Architecture
```

Comment:
Like it!

Reply:
lol, i'm confused. I didn't write any of this..